web/themes/custom/pineapple/sass/layouts/nodes/_recipe.scss

```diff
@@ -81,14 +87,18 @@
       }
 
       .author-quote {
-        padding: $unit *2;
+        padding: 0 $unit *4;
         line-height: 1.5em;
         max-height: 8.2em;
         text-align: center;
         overflow: hidden;
         display: -webkit-box;
         -webkit-line-clamp: 5;
         -webkit-box-orient: vertical;
+
+        @include breakpoint($lap) {
+          padding: $unit *2;
```

Comment:
There's too much space/uneven between Image \ name \ and bio teaser text

Reply:
Fixed.